types/react-map-gl/index.d.ts

```diff
@@ -97,8 +97,8 @@ export interface MapControllerOptions {
 }
 
 export interface ViewportProps {
-    width: number;
-    height: number;
+    width: number | string;
+    height: number | string;
```

Comment:
I don't think this is correct, these should be numbers. Take a look at https://github.com/uber/react-map-gl/blob/master/src/utils/map-state.js

Reply:
Thanks for the review @fnberta 

See [This example](https://github.com/uber/react-map-gl/blob/master/examples/exhibit-browserify/app.js) or (these prop-types](https://github.com/uber/react-map-gl/blob/f2bc61b6184d99c3ffa7267373b16e380dd600ba/src/components/static-map.js#L56). Using a string allows for setting width and height to '100vw' and '100vh' respectively.

Reply:
Yes, but these examples are covered by the `MapboxProps` definition which includes:
```
width: number | string;
height: number | string;
```
Or am I missing something?

Reply:
I see. Thanks for the clarification.